server/conf/messages.en-US

```diff
@@ -340,3 +343,12 @@ validation.textTooShort=Must contain at least {0} characters.
 
 # Validation errors that appear if an admin leaves a multi option question blank.
 adminValidation.multiOptionEmpty=Multi option questions cannot have blank options.
+
+#-------------------------------------------------------#
```

Comment:
The same changes in the messages.en-US file should also be done to the messages files.

Reply:
I tried it. Did I do it right?